hudi-common/src/main/java/org/apache/hudi/metadata/HoodieTableMetadataUtil.java

```diff
@@ -1143,7 +1114,6 @@ private static List<HoodieRecord> convertMetadataToRollbackRecords(HoodieRollbac
                                                                      HoodieTableMetaClient dataTableMetaClient) {
     Map<String, Map<String, Long>> partitionToAppendedFiles = new HashMap<>();
     processRollbackMetadata(rollbackMetadata, partitionToAppendedFiles);
-    reAddLogFilesFromRollbackPlan(dataTableMetaClient, instantTime, partitionToAppendedFiles);
```

Comment:
why remove this

Reply:
Please check discussion regarding it in https://github.com/apache/hudi/pull/13007#discussion_r2017704147